packages/superset-ui-legacy-plugin-chart-map-box/src/ScatterPlotGlowOverlay.jsx

```diff
@@ -151,7 +150,7 @@ class ScatterPlotGlowOverlay extends React.PureComponent {
           ctx.beginPath();
           if (location.get('properties').get('cluster')) {
             let clusterLabel = clusterLabelMap[i];
-            const scaledRadius = roundDecimal(Math.pow(clusterLabel / maxLabel, 0.5) * radius, 1);
+            const scaledRadius = roundDecimal((clusterLabel / maxLabel) ** 0.5 * radius, 1);
```

Comment:
This is fine, but just as food for thought, this instance of ** is a slightly less friendly than the first use on line 63. In this use case, the reader has to consiously think about order of operations, whereas Math.pow(...) makes that pretty explicit.

Reply:
bringing back `Math.pow`